<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="step_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>680</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Step by step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>640</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>開啟文字檔</string>
   </property>
  </widget>
  <widget class="QPushButton" name="output_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>680</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>輸出文字檔</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>640</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清除畫布</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Run_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="mouse_location">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mouse_location</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="canvas">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>610</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>120</width>
     <height>451</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>118</width>
      <height>449</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>滑鼠位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>點個數</string>
   </property>
  </widget>
  <widget class="QLabel" name="point_num">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="next_data">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>640</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>讀入下一筆資料</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reload_output">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>讀取輸出檔案</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>輸入的點座標</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="output_area">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>150</y>
     <width>120</width>
     <height>451</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>118</width>
      <height>449</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>程式輸出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
